llvm/lib/Target/PowerPC/PPCRegisterInfo.td

```diff
@@ -270,12 +270,15 @@ def CR7 : CR<7, "cr7", [CR7LT, CR7GT, CR7EQ, CR7UN]>, DwarfRegNum<[75, 75]>;
 
 // Link register
 def LR  : SPR<8, "lr">, DwarfRegNum<[-2, 65]>;
-//let Aliases = [LR] in
```

Comment:
Do you know why it was commented out?

Reply:
see https://github.com/llvm/llvm-project/commit/44dbdbe5cf5e

I guess it is because Aliases was not a member of Register at that time.